R/post_test_analysis.R

```diff
@@ -562,12 +551,15 @@ print.summary.GeoLift <- function(x, ...) {
       "\n\n##################################",
       "\n#####     Model Weights      #####\n",
       "##################################\n",
+      "\n* Prognostic Function: ", toupper(x$progfunc), "\n",
       "\n* Model Weights:"
-    ))
-    for (row in 1:nrow(x$weights)) {
-      if (abs(round(as.double(x$weights$weight[row]), 4)) >= 0.0001) {
-        message(paste0(" * ", x$weights$location[row], ": ", round(x$weights$weight[row], 4)))
-      }
+    )
+    message(test_statistics, balance_statistics)
+  }
+  sorted_weights <- dplyr::arrange(x$weights, dplyr::desc(abs(weight)))
```

Comment:
This fails because `weight` is not defined. You should change it to:
`sorted_weights <- dplyr::arrange(x$weights, dplyr::desc(abs(x$weights$weight))).

Remember to run devtools::check(vignettes = FALSE) when you finish your PR to check for any errors. These types of things should come up.